guides/common/modules/proc_comparing-content-view-versions.adoc

```diff
@@ -4,7 +4,9 @@
 Use this procedure to compare Content View Version functionality for {Project}.
 
 .Procedure
-* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click *Compare*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
+. Select the content view version you want to compare.
```

Comment:
I myself don't know why we keep this capitalized but it's probably a good idea to keep it consistent. Also, is **version** supposed to be in singular? It is plural in the current revision.
```suggestion
. Select the Content View version you want to compare.
```

Reply:
I may need to submit a PR down the road where all mentions of content view(s) in the Managing Content guide are capitalized because I know it's not consistent throughout which is why I had it lower case here.